app/lib/audit/models.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+import '../account/models.dart';
+import '../shared/datastore.dart' as db;
+import '../shared/utils.dart' show createUuid;
+
+@db.Kind(name: 'AuditLogRecord', idType: db.IdType.String)
+class AuditLogRecord extends db.ExpandoModel<String> {
+  @db.DateTimeProperty(required: true)
+  DateTime created;
+
+  @db.DateTimeProperty()
+  DateTime expires;
+
+  @db.StringProperty(required: true)
+  String kind;
+
+  @db.StringProperty(required: true)
+  String agent;
+
+  @db.StringProperty(required: true)
+  String summary;
```

Comment:
```suggestion
  /// Textual summary of the action that was performed in **markdown**.
  ///
  /// This summary should contain a full explanation of the action that was taken,
  /// including who authorized it, which user or resource was subject of the action.
  /// Users are identified by the email associated with their account at the point-in-time
  /// when the action was taken.
  ///
  /// The following properties are intended for presentation in a table when rendering an audit-log:
  ///  * [created]
  ///  * [kind]
  ///  * [agent] (left as `<deleted-user>` if user no longer exists)
  ///  * [summary]
  ///
  /// This can only use simple markdown formatting no block structures.
  @db.StringProperty(required: true, indexed: false)
  String summary;
```

I think there is somewhere else where we use "simple markdown without block structures", I think it's in publisher descriptions.